spec/prog/vm/nexus_spec.rb

```diff
@@ -976,7 +976,7 @@ def requested_disk_size(st)
         expect(strand).to eq st
         expect(storage_volumes).to eq [{size_gib: 1}]
       end.and_return(true)
-      expect(nx).to receive(:frame).and_return("attempt" => 1, "storage_volumes" => [{"size_gib" => 1}]).at_least(:once)
+      expect(nx).to receive(:frame).and_return("attempt" => 1, "assemble_storage_volumes" => [{"size_gib" => 1}]).at_least(:once)
```

Comment:
I suppose our mistake here can be seen as a problem of using these unschema'd data structures to too great an extent. Normally, expect's verified double behavior checks for method existence, that would have spared us from a bogus mock while also sparing us from the speed (and coupling that can blow up combinatorics) of integration testing.

When dealing with straight up (parsed) json, we're without any support.